test/jdk/java/awt/Frame/DisposeTest.java

```diff
@@ -74,8 +76,8 @@ private static void test() {
         BufferedImage bi = robot.createScreenCapture(backgroundFrameBounds);
         int redPix = Color.RED.getRGB();
 
-        for (int x = 0; x < bi.getWidth(); x++) {
-            for (int y = 0; y < bi.getHeight(); y++) {
+        for (int x = PIXEL_OFFSET; x < bi.getWidth() - PIXEL_OFFSET; x++) {
+            for (int y = PIXEL_OFFSET; y < bi.getHeight() - PIXEL_OFFSET; y++) {
```

Comment:
It might be better to increase the size of the background frame so that its edge does not appear under the tested frame.

Reply:
Currently the backroundFrame is 200x200 and testFrame is 100x100. The difference of 100 seems sufficient for this case.